src/widgets/cards-list/index.tsx

```diff
@@ -0,0 +1,223 @@
+import Grid from '@mui/material/Unstable_Grid2';
+import { Card } from '~/entities';
+import { AddCardButton } from '~/features';
+
+export const CardsList = ({
+  items = cards,
+  onClickCard,
+  onClickLike,
+  onClickAddCard,
+}) => {
+  return (
+    <Grid
+      container
+      spacing={2}
+      columns={{ xs: 12, sm: 12, md: 12 }}
```

Comment:
Достаточно указать для xs количество колонок. А остальные нужны если оно будет меняться на более высоких разрешениях.
```suggestion
      columns={{ xs: 12 }}
```